composer.json

```diff
@@ -90,7 +90,7 @@
     },
     "require-dev": {
         "allure-framework/allure-phpunit": "^2",
-        "dealerdirect/phpcodesniffer-composer-installer": "^0.7",
+        "dealerdirect/phpcodesniffer-composer-installer": "^0.7 || ^1.0",
```

Comment:
As same as for codding-standard repo PR I believe that `"^1.0"` will be enough

Reply:
There's nothing here that's *not* compatible with `^0.7.0`. I've specifically kept this because I expect other packages may take a while to upgrade / allow the new version. If there's something that requires `^0.7`, it'll work today, but not after merging if we change this to `^1.0` only. Listing both supported versions means that interoperability with other packages is maintained; Composer will pick the "best" version, which is typically the highest mutually-supported version. If we use `^1.0` only here, then we're forcing others to upgrade their packages in order for their coding standard to be used in conjunction with this product.

Reply:
Make sense